lib/gosu/compat.rb

```diff
@@ -0,0 +1,187 @@
+# This whole file is about backward compatibility
+
+# First - define some methods to tell the world about their old code
+module Gosu
+  module Deprecate
```

Comment:
Hmm, `Gem::Deprecate` works the same, but do you know if there is an advantage in having a separate module for deprecations? Can't we just define all the methods on `Gosu`? Then there'd be one less exception in `test_interface`.